style.css

```diff
@@ -551,9 +636,199 @@ button.interaction:disabled {
+    border: none;
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+    width: 100%;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: flex-start;
+  }
+
+  .h3-title {
+    margin-left: 6%;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
+
+  hr {
+    visibility: visible;
+    height: 1px;
+    border-width: 0;
+    background-color: #212121;
+    width: 52%;
+    margin-right: 15.5%;
+  }
+
+  .frame44 {
+    height: fit-content;
+    grid-row: 2 / 3;
+    grid-column: 1 /2;
+    width: 100%;
+    padding-bottom: 0;
+  }
+
+  .featured-img {
+    width: 90%;
+    height: fit-content;
+    display: grid;
+    grid-template-rows: 0.4fr 1fr;
+    grid-template-columns: 1fr 0.8fr;
+    margin-left: 0.5%;
+    padding-left: 0;
+  }
+
+  .featured-img > img {
+    grid-row: 1 / 2;
+    grid-column: 1 / 2;
+    width: 100%;
+    height: fit-content;
+    align-self: start;
+  }
+
+  .right-block {
+    grid-row: 1 / 2;
+    grid-column: 2 / 3;
+    width: 100%;
+    height: fit-content;
+    gap: 30px;
+  }
+
+  .block-h2 {
+    width: 70%;
+  }
+
+  .block-p {
+    width: 70%;
+    height: fit-content;
+  }
+
+  .block-nav {
+    width: 100%;
+  }
+
+  .frame43 {
+    width: 100%;
+    height: auto;
+    grid-row: 2 / 3;
+    grid-column: 1 / 3;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+  }
+
+  .bg-printing {
+    background-image: url("images/bg-one.svg");
+    display: flex;
+    flex-direction: column;
+    justify-content: flex-end;
+  }
+
+  .hide {
+    visibility: hidden;
+  }
+
+  .bg-availability {
+    background-image: url("images/bg-two-five.svg");
+  }
+
+  .background {
+    background-image: url("images/bg-three-six.svg");
+  }
+
+  .bg-print {
+    background-image: url("images/bg-four.svg");
+  }
+
+  .btn-hide {
+    visibility: hidden;
+  }
+
+  .card-group {
+    display: flex;
+    flex-direction: column;
+    gap: 0;
+    padding-top: 110px;
+    padding-left: 10px;
+  }
+
+  .about-me {
+    margin-top: 0;
+    width: 100%;
+    height: fit-content;
+    padding: 7vh 0 0 7vw;
+    background-image: url("images/Frame.svg");
+    background-position: 100% 12%;
+  }
+
+  .about-me > h2 {
+    margin-bottom: 1vh;
+  }
+
+  .about-me > p {
+    width: 45%;
+    margin-top: 0;
+    margin-bottom: 1vh;
+  }
+
+  .about-me > button {
+    margin-top: 1vh;
+  }
+
+  #lang {
+    width: 90%;
+    margin-top: 10vh;
+    margin-bottom: 10vh;
+    display: flex;
+    justify-content: flex-start;
+    padding-left: 0;
+    border-top: 1px solid #fff;
+  }
+
+  #lang > ul {
+    width: 100%;
+    padding-left: 0;
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    gap: 20%;
+    margin: 0 7vw;
+  }
+
+  #language {
+    background-position: 0 15%;
+  }
+
+  #language > h3 {
+    margin-left: 25px;
+  }
+
+  #framework {
+    background-position: 5% 14%;
+  }
```

Comment:
At around screen width 800px, a part of the skills & languages dissappears. Please check and fix that to make in conform to the design. 

 Your design            |  The design template
:-------------------------:|:-------------------------:
![Screenshot from 2022-11-12 06-58-13](https://user-images.githubusercontent.com/65233860/201461836-27285722-cc43-4ee7-b69b-c6dcdf8bf8b7.png)  |  ![Screenshot from 2022-11-12 07-00-14](https://user-images.githubusercontent.com/65233860/201461849-3661e1ba-a1d2-4d68-b49e-c20795bda895.png)